nordicsemi/lister/windows/structures.py

```diff
@@ -23,16 +23,18 @@
 """
 
 import ctypes
+from ctypes.wintypes import DWORD, BYTE, WORD
+from serial.win32 import ULONG_PTR
 
 _ole32 = ctypes.WinDLL('ole32')
 
 
 class _GUID(ctypes.Structure):
     _fields_ = [
-        ('Data1', ctypes.c_uint32),
```

Comment:
Unless there was a particular reason for switching to WORD, DWORD, etc, I would prefer to use the old and IMO more descriptive types uint16, uint32. They are also more in line with Nordic's coding standard.